pkg/kubelet/prober/worker.go

```diff
@@ -154,6 +161,7 @@ probeLoop:
 		case <-w.stopCh:
 			break probeLoop
```

Comment:
should the `manualTriggerCh` be closed here? What's the risk of keeping it not closed ever?

Reply:
well, we could say the same for the `stopCh` no?

Reply:
yes. Not sure what the downside will be to not close it.

Reply:
I leave it like that, let's see what the approvers have to say.